packages/components/select-v2/src/useSelect.ts

```diff
@@ -301,7 +301,13 @@ const useSelect = (props: ISelectV2Props, emit: SelectEmitFn) => {
     const padding =
       Number.parseFloat(style.paddingLeft) +
       Number.parseFloat(style.paddingRight)
-    ctx.font = style.font
+    const fontParts = style.font.split(' ')
+    ctx.font = [
+      'bold',
+      ...fontParts.filter(
+        (p) => !p.match(/(bold|normal|lighter|bolder|\d{3})/)
+      ),
+    ].join(' ')
```

Comment:
I think it seems more intuitive to write it this way.
```ts
ctx.font = `bold ${style.font.replace(new RegExp('\\b' + style.fontWeight + '\\b'), '')}`
```

Reply:
When you set font-weight `bold|normal|lighter|bolder`, the computed style will be a number. So that's not correct.

Reply:
If the `font-weight` value returned by `style.font` is a number, then `style.fontWeight` should also be returned as a number.

Reply:
Yes, but the font-weight returned by style.font is also 700.

![image](https://github.com/user-attachments/assets/893236fc-667a-414a-a452-6dc597972f69)

Reply:
Ok, i see.